packages/generator-spectral/src/generators/formats/readers/openapi/actions.ts

```diff
@@ -78,9 +78,7 @@ const buildAction = (
     | OpenAPIV3_1.ParameterObject
   )[] = []
 ): Action => {
-  if (!operation.operationId) {
-    throw new Error(`Failed to find operationId for ${path} ${verb}`);
-  }
+  const operationName = camelCase(operation.operationId || `${verb} ${path}`);
```

Comment:
Can the `key` have a space like that? I don't recall if we use that `operationName` for variable names.

Reply:
It all gets run through `camelCase` at the end so a `POST /foo/bar/baz/{id}` becomes `postFooBarBazId` - not great but it should at least successfully generate the component.

Reply:
Ah; perfect!